engine/tests/phpunit/unit/Elgg/Views/ViewRenderingTestCase.php

```diff
@@ -74,6 +92,7 @@ public function assertViewOutput($expected, $view, $view_vars = [], $viewtype =
 	 */
 	public function testCanRenderViewWithEmptyVars($view, $viewtype) {
 		if (!elgg_view_exists($view, $viewtype)) {
+			dump($view);
```

Comment:
is this left over debug code?

Reply:
Oops, yes

Reply:
if you could remove that, i'll merge this

Reply:
done